recipes/ovs_offload/Testlib.py

```diff
@@ -45,8 +45,8 @@ def find_tc_rule(self, host, nic, src_mac, dst_mac, proto='.*', action='.*', tun
             return None
 
         # find rule
-        pat_filter = r"^protocol %s pref .* dst_mac %s\n\s*src_mac %s\n.*\n\s*action order \s*\d+\s*:\s* %s" % (
-                proto, str(dst_mac).lower(), str(src_mac).lower(), action)
+        pat_filter = r"^protocol .* pref .* dst_mac %s\n\s*src_mac %s\n\s*eth_type %s\n.*\n\s*action order \s*\d+\s*:\s* %s" % (
```

Comment:
Instead of .* after protocol use \w+ so we will match a single word and not any length of string

Reply:
You what you can leave it .* Doesn't tmatter so much here

Reply:
Ok. I will keep it as you want.

Reply:
Ok. I will leave it as you want.